### ISSUE TITLE
Horizontal scrolling with mouse wheel wrong behavior

### ISSUE THREAD

**Issue by sgabenov** [ONLYOFFICE/DesktopEditors#1716]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

macOS 10.12 or higher

### Version information

8.2.0

### Expected Behavior

In spreadsheet when i use second mouse wheel () for horizontal scrolling the sheet table should be scrolled only horizontal. 

### Actual Behavior

When i start scrolling with second mouse wheel for horizontal scrolling - both scrolling directions are executed (down + right when scrolling right with second wheel or up + left when scrolling left with second mouse wheel). The behaviour is like i am using both mouse wheels in parallel. I have 2 mac and 2 different mouses with same model - the behavior is the same, so i am not suspecting that the problem is in mouse. Also, i do not have this problem when scrolling in Chrome browser 

### Reproduction Steps

create table with 100 rows and 100 colums. Start scrolling down with second mouse wheel 

### Additional information

i use mouse Logi MX master 3s with this behavior

**Comment by Rita-Bubnova** [comment]
Duplicate of https://github.com/ONLYOFFICE/DocumentServer/issues/536.
